components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -19,38 +19,35 @@ IAM v2 is a beta release and its functionality is subject to change during this
 This is an opt-in only feature during the beta period.
 Chef Automate users will not be automatically upgraded.
 
-We designed IAM v2 to leave your v1 policy data untouched during your upgrade to v2; however, you can choose to not port over v1 policies by using the provided 
-`--skip-policy-migration` flag with the upgrade command.
+We designed IAM v2 to leave your v1 policy data untouched during your upgrade to v2; however, you can choose to not port over v1 policies by using the provided `--skip-policy-migration` flag with the upgrade command.
 
 Whether or not you migrate your v1 policies, if at any time you decide to opt back out of the beta and revert to v1, your original v1 policies will still be intact.
-Reverting to v1, however, will remove any new v2 policies or roles created while using IAM v2.
-Note that this applies only to policies and roles in this beta period. Users, teams, and tokens are currently shared between v1 and v2, but these are slated for separation before the full v2 release.
+Reverting to v1 will remove any new v2 data created while using IAM v2 (policies, roles, projects, rules).
```

Comment:
in general, i think we should refer to `rules` as `project rules` to be more specific.

Reply:
yes, agree!